app/components/primer/blankslate_component.html.erb

```diff
@@ -1,13 +1,13 @@
 <%= render Primer::BaseComponent.new(**@system_arguments) do %>
-  <% if @icon.present? %>
+  <% if spinner.present? %>
+    <%= render spinner.component %>
+  <% elsif @icon.present? %>
```

Comment:
One small change here @joelhawksley, which I think is a bug fix. You can't have a `spinner`, `@icon`, and a `@image_src` at the same time.

Reply:
👍🏻  I like it.